operations/rack_resiliency/Setup_of_Rack_Resiliency.md

```diff
@@ -7,7 +7,7 @@ Specifically, the setup is done by the `rack_resiliency_for_mgmt_nodes.yml` Ansi
 repository.
 
 If Rack Resiliency is not enabled, then this playbook will do nothing.
-See [Enabling Rack Resiliency](Enabling_Rack_Resiliency.md) for details on
+See [Enabling Rack Resiliency](README.md#enable-and-configure) for details on
```

Comment:
If you are altering the link destination, the link text also needs to be altered to match the new destination (since this new destination is not called `Enabling Rack Resiliency`)